src/components/SetupFunders/SetupFundersTest.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
```

Comment:
This file needs to be renamed to SetupFunders.test.tsx, as it's not being picked up by Jest. This had been updated in the latest hygen templates.